packages/sheets-ui/src/controllers/editor/editing.render-controller.ts

```diff
@@ -845,6 +846,13 @@ export class EditingRenderController extends Disposable implements IRenderModule
             this._editorBridgeService.interceptor.getInterceptPoints().AFTER_CELL_EDIT_ASYNC
         )(Promise.resolve(cell), context);
 
+        const finalCellValue = finalCell?.p?.body?.dataStream.trim();
```

Comment:
It is recommended to judge in getCellDataByInput and add unit test

Reply:
I have reversed my previous modifications. I chose to implement the relevant logic in `getCellDataByInput` and modified the wrong test case. Since the complete functional implementation still requires the addition of automatic recognition format code logic, I don't plan to add new unit tests for the time being.